### ISSUE TITLE
Edit the parent directory of the current buffer

### ISSUE THREAD

**Issue by alexherbo2** [TeddyDD/kakoune-edit-or-dir#9]
Recently @FlyingWombat suggested me a command to edit the parent directory of the current buffer, which could be `edit-or-dir` with no argument.

The relevant issue can be found [here](https://github.com/alexherbo2/explore.kak/pull/1).

**Comment by TeddyDD** [comment]
It's available as `e ..`. It works but there is error (plugin searches for previous directory, I should probably wrap it in try block).

**Comment by alexherbo2** [comment]
`..` is relative to the current directory, it’s different from the buffer file directory.

**Comment by TeddyDD** [comment]
Ah, yeah, that's true. It's something to think about after I fix #3 (If I ever fix this, escaping issues are making me mad every time I try :laughing: )

For now this should work:
```
define-command ep %{
   edit-or-dir %sh{ dirname $kak_bufffile }
}
```

**Comment by alexherbo2** [comment]
At [L5](https://github.com/TeddyDD/kakoune-edit-or-dir/blob/master/rc/edit-or-dir.kak#L5) you can `set -- "$buffer_file_directory"` when `$#` is 0.
